### PR TITLE
added show/hide introtext parameter to mod_articles_news

### PR DESCRIPTION
**Pull request by coolcat-creations**
#### Summary of Changes

Added a show_introtext parameter to the newsflash module
#### Testing Instructions

Before applying the patch publish a Newsflashmodule 
1) After applying the patch look if the introtext is still shown as default
2) Set Introtext to Hide -> Introtext should be hidden
4) Set Introtext to Show -> Introtext should be shown
#### Documentation Changes Required

image

### CODE REVIEW

File: language/en-GB/en-GB.mod_articles_news.ini [hunk 1]
```diff
@@ -23,6 +23,8 @@ MOD_ARTICLES_NEWS_FIELD_SEPARATOR_DESC="Show separator after last Article."
 MOD_ARTICLES_NEWS_FIELD_SEPARATOR_LABEL="Show Last Separator"
 MOD_ARTICLES_NEWS_FIELD_TITLE_DESC="Show or hide the Article title."
 MOD_ARTICLES_NEWS_FIELD_TITLE_LABEL="Show Article Title"
+MOD_ARTICLES_NEWS_FIELD_SHOWINTROTEXT_DESC="Select Show if you would like the introtext to be displayed."
```
Comment (jeckodevelopment) [inline language/en-GB/en-GB.mod_articles_news.ini]: For consistency with the string `MOD_ARTICLES_NEWS_FIELD_TITLE_DESC` you could consider to change the description in this way:

> Show or hide the introtext

Comment (jeckodevelopment) [inline language/en-GB/en-GB.mod_articles_news.ini]: in `administrator/language/en-GB/en-GB.com_content.ini`
we use "Intro Text" as separate words

**Comment by zero-24** [comment]
please have a look here: https://github.com/designbengel/joomla-cms/pull/4 the option needs to be enabled by default for B/C :smile:

**Comment by designbengel** [comment]
@zero-24 i enabled it by default ?

**Comment by designbengel** [comment]
@jeckodevelopment that´s why i copied the text/translation and the parameter name from the mod_articles_category ?
I can change it of course, but should i change it then in mod_articles_category also ?

**Comment by zero-24** [comment]
@designbengel 

>@zero-24 i enabled it by default ?

https://github.com/joomla/joomla-cms/pull/11564/files#diff-1b513a4609e09b6bd5c4b989a6541998R107

default is = "0"?

**Comment by designbengel** [comment]
@zero-24 sorry seems i enabled it without comitting :-D (facepalm) ;)

**Comment by jeckodevelopment** [comment]
@designbengel let's wait for @brianteeman 's opinion :)

**Comment by brianteeman** [comment]
Can you change the label to 
"Show Intro Text"
and the description to
"Show or hide the article intro text."

I've been trying to standardise on that and have an RTC pull request to fix most of the remaining instances



(note to self to review our usage of "intro text" and "introtext"

**Comment by 1apweb** [comment]
I have tested this item :red_circle: unsuccessfully on 59efa711f0c72507f41a108b54692cd6a8b712f9<br /><br />Hi,
Show: the whole article appears
Hide: the article does not appear<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11564">issues.joomla.org/joomla-cms/11564</a>.</sub>

**Comment by zero-24** [comment]
@1apweb is that not the expected behavior? What did not work?

**Comment by 1apweb** [comment]
I have tested this item :white_check_mark: successfully on 59efa711f0c72507f41a108b54692cd6a8b712f9<br /><br />Sorry, its ok now.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11564">issues.joomla.org/joomla-cms/11564</a>.</sub>

**Comment by truptikagathara** [comment]
I have tested this item :white_check_mark: successfully on 8c0e15a44b53bd8104f5aff543296cff9238b17d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11564">issues.joomla.org/joomla-cms/11564</a>.</sub>

**Comment by designbengel** [comment]
@1apweb & @truptikagathara thank you for testing

**Comment by mikeveeckmans** [comment]
I have tested this item :white_check_mark: successfully on 8c0e15a44b53bd8104f5aff543296cff9238b17d<br /><br />patch tested and works<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11564">issues.joomla.org/joomla-cms/11564</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11564">issues.joomla.org/joomla-cms/11564</a>.</sub>

**Comment by yvesh** [comment]
I have tested this item :white_check_mark: successfully on 8c0e15a44b53bd8104f5aff543296cff9238b17d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11564">issues.joomla.org/joomla-cms/11564</a>.</sub>

**Comment by designbengel** [comment]
Thank you @yvesh @mikeveeckmans @brianteeman @jeckodevelopment @1apweb @truptikagathara @zero-24 - for supporting my first PR ;)

**Comment by jeckodevelopment** [comment]
you're welcome @designbengel , thank you for your PR :)

**Comment by wilsonge** [comment]
Merged to the 3.7.x branch with https://github.com/joomla/joomla-cms/commit/84b6c702dc9d79bc25e8cd659f66ab41fc578d1d. Thanks Elisa!

**Pull request closed without merging**